include/cuco/detail/utils.cuh

```diff
@@ -17,6 +17,7 @@
 
 #include <cuco/detail/bitwise_compare.cuh>
 
+#include <cuda/std/array>
```

Comment:
```suggestion
```
suggestion: A leftover to be removed. could you also remove the inclusion of `#include <cuda/std/cmath>` as well since we no longer invoke ` cuda::std::abs` in this file?